app/grandchallenge/challenges/models.py

```diff
@@ -102,20 +105,51 @@
 class ChallengeSet(models.QuerySet):
     def with_available_compute(self):
         return self.annotate(
-            approved_compute_costs_euro_millicents=(
+            complimentary_compute_costs=(
                 Sum(
                     "invoices__compute_costs_euros",
                     filter=Q(
-                        invoices__payment_status__in=[
-                            PaymentStatusChoices.COMPLIMENTARY,
-                            PaymentStatusChoices.PAID,
-                        ]
+                        invoices__payment_type=PaymentTypeChoices.COMPLIMENTARY
                     ),
                     output_field=models.PositiveBigIntegerField(),
                     default=0,
                 )
+            ),
+            prepaid_compute_costs=(
+                Sum(
+                    "invoices__compute_costs_euros",
+                    filter=Q(invoices__payment_type=PaymentTypeChoices.PREPAID)
+                    & Q(invoices__payment_status=PaymentStatusChoices.PAID),
```

Comment:
Separate filters are unnecessary in this case

```suggestion
                    filter=Q(invoices__payment_type=PaymentTypeChoices.PREPAID, invoices__payment_status=PaymentStatusChoices.PAID),
```